frontend/src/app/modules/bim/bcf/bcf-buttons/bcf-import-button.component.ts

```diff
@@ -28,7 +28,6 @@
 
 import {Component} from '@angular/core';
 import {I18nService} from 'core-app/modules/common/i18n/i18n.service';
-import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
 import {CurrentProjectService} from "core-components/projects/current-project.service";
 import {BcfPathHelperService} from "core-app/modules/bim/bcf/helper/bcf-path-helper.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/be03a9ed2bd960efe9bf08e68a4088034cb73cd9/frontend/src/app/modules/bim/bcf/bcf-buttons/bcf-import-button.component.ts#L33"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>